app/web_ui/src/routes/(app)/docs/rag_configs/[project_id]/table_rag_config_row.svelte

```diff
@@ -78,6 +77,14 @@
   function open() {
     goto(`/docs/rag_configs/${project_id}/${rag_config.id}/rag_config`)
   }
+
+  $: chunk_size = rag_config.chunker_config.properties.chunk_size as number
```

Comment:
no casting is a pretty firm rule, it will burn us if we add types with different properties (rendering "undefined" or raising).

Just check the the props exist in the format_chunking for now.

Later we should use typed dicts for all properties (Daniel just did this for MCP, it's great, makes the UI code much cleaner)